vhost-device-sound/src/audio_backends/pipewire/test_utils.rs

```diff
@@ -41,9 +59,9 @@ impl DbusSession {
 
 impl Drop for DbusSession {
     fn drop(&mut self) {
-        println!("INFO: Killing Dbus session {}", self.child.id());
+        eprintln!("INFO: Killing Dbus session {}", self.child.id());
```

Comment:
I'd change these prints in a different patch or at least mention in the description, but they are just test stuff, so, not a strong opinion.

Reply:
+1

Reply:
Ended up updating the commit description